### PR TITLE
add git actions

### PR DESCRIPTION
**Pull request by utopianet**
change：
One files have been added
Files：
build.yml：Build backend and frontend
Actions：
Checkout
Set up JDK
Set up Node.js
Build backend by maven
Build frontend by node.js

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# This workflow will do a clean install of node dependencies, cache/restore them, build the source code and run tests across different versions of node
```
Comment (peacewong) [inline .github/workflows/build.yml:1]: Need to add license

**Comment by casionone** [comment]
Maybe the action name “Node.js CI” is not accurate. 
It would be great to start a discussion email. such as  [https://lists.apache.org/thread/4m26q8b6st66rhthojr2wc9r3bxbt066](https://lists.apache.org/thread/4m26q8b6st66rhthojr2wc9r3bxbt066)

**Comment by casionone** [comment]
LGTM

**Review by peacewong — APPROVED** [review]
LGTM.

**Pull request merged**